package Exception_Coding_Assessment;

public class Challenge3_DifferentTypeExceptions {
    //Write a program that catches both `ArithmeticException` and `ArrayIndexOutOfBoundsException`.
    //Input=Division by zero in array operation
    //Output=Exception caught: ArithmeticException or ArrayIndexOutOfBoundsException
}
